api-spec/extensions/sort/README.md

```diff
@@ -2,7 +2,29 @@
 
 **Extension [Maturity Classification](../../../extensions/README.md#extension-maturity): Pilot**
 
-The STAC search endpoint, `/stac/search`, by default returns results in descending order using the datetime property. The sort API extension adds a new parameter, `sort` that allows the user to define fields to sort results by. Only properties may be used to sort results. The syntax for the `sort` parameter is:
+By default, the STAC search endpoint `/stac/search` returns results in no specified order.  Whatever order the results are in is up to the implementor, and will typically default to an arbitrary that is fastest for the underlying data store to retrieve results.
+ 
+ The Sort API Extension adds a new field, `sort`, that allows the user to define fields by which to sort results. Only string, numeric, and datetime attributes of Item (`id` and `collection` only) or Item Properties (any attributes) may be used to sort results.  It is not required that implementations support sorting over all attributes, but implementations should return an error when attempting to sort over a field that does not support sorting. 
+
+Two values for direction are supported: "asc" (ascending) or "desc" (descending). If the direction is not specified, the value is ascending. The `sort` value is an array, so multiple sort fields can be defined which will be used to sort the data in the order provided (e.g., first by `datetime`, then by `eo:cloud_cover`).
+
+## GET or POST Form
+
+When calling `/stac/search` using GET or POST with `Content-Type: application/x-www-form-urlencoded` or `Content-Type: multipart/form-data`, the semantics are the same, except the syntax is a single parameter `sort` with a comma-separated list of "<name>|<direction>" definitions.  The sort order must be specified.
+
+Examples of `sort` parameter:
+
+    GET /stac/search?sort=created|asc,id|desc
```

Comment:
What about replacing `|` with `;` ?  Like `GET /stac/search?sort=eo:cloud_cover;desc`.  `|` may be ambiguous (expressing this or that property, say)